src/multisig.js

```diff
@@ -383,3 +380,65 @@ export function multisigPublicKeys(multisig) {
 export function multisigAddress(multisig) {
   return multisig.address;
 }
+
+/**
+ * Return the braid details (if known) for a given `Multisig` object.
+ *
+ * @param {module:multisig.Multisig} multisig the `Multisig` object
+ * @returns {string} the braid details
+ * @example
+ * import {
+ *   generateBraidFromExtendedPublicKeys,
+ *   generateMultisigFromPublicKeys, MAINNET, P2SH,
+ *   braidConfig,
+ * } from "unchained-bitcoin";
+ * const multisig = generateMultisigFromPublicKeys(MAINNET, P2SH, 2, "03a...", "03b...", "03c...");
+ * console.log(braidConfig(multisig)); // null, unknown
+ *
+ * const braid = generateBraidFromExtenedPublicKeys(MAINNET, P2SH, {{'xpub...', bip32path: "m/45'/0'/0'"}, {'xpub...', bip32path: "m/45'/0/0"}, {'xpub...', bip32path: "m/45'/0/0"}}, 2);
+ * const multisig = braid.deriveMultisigByPath("0/0");
+ * console.log(braidConfig(multisig)); // {network: mainnet, addressType: p2sh, extendedPublicKeys: {...}, requiredSigners: 2}}
+ */
+export function braidDetails(multisig) {
+  return multisig.braidDetails ? multisig.braidDetails : null;
+}
+
+/**
+ * Return the bip32Derivation (if known) for a given `Multisig` object.
+ *
+ * @param {module:multisig.Multisig} multisig the `Multisig` object
+ * @param {number} index the index to generate at
+ * @returns {[Object[]]} the bip32Derivation includes all paths to all pubkeys in the multisig object
+ * @example
+ * import {
+ *   bip32Derivation,
+ *   generateBraidFromExtendedPublicKeys,
+ *   generateMultisigFromPublicKeys, MAINNET, P2SH,
+ *   braidConfig,
+ * } from "unchained-bitcoin";
+ * const multisig = generateMultisigFromPublicKeys(MAINNET, P2SH, 2, "03a...", "03b...", "03c...");
+ * console.log(bip32Derivation(multisig, 0)); // null, unknown
+ *
+ * const braid = generateBraidFromExtendedPublicKeys(MAINNET, P2SH, {{'xpub...', bip32path: "m/45'/0'/0'"}, {'xpub...', bip32path: "m/45'/0/0"}, {'xpub...', bip32path: "m/45'/0/0"}}, 2);
+ * const multisig = braid.deriveMultisigByIndex("0");
+ * console.log(bip32Derivation(multisig, 0)); // {
+ *   {masterFingerprint: Buffer('1234..', 'hex'), path: "m/45'/0'/0'/0/0", pubkey: Buffer.from("02...", 'hex')}
+ *   {masterFingerprint: Buffer('3453..', 'hex'), path: "m/45'/0/0/0/0", pubkey: Buffer.from("03...", 'hex')}
+ *   {masterFingerprint: Buffer('1533..', 'hex'), path: "m/45'/0/0/0/0", pubkey: Buffer.from("02...", 'hex')}
+ * }
+ */
+export function bip32Derivation(multisig, index) {
+  const config = JSON.parse(braidDetails(multisig));
```

Comment:
Why the `JSON.parse` here? 

This kinda makes me feel like multisig should be a class or braidDetails? i'm not sure but this feels like a lot of steps to do something that could be common and should be more easily accessible. This feels like a nicer api:

```
const multisig = new Multisig({...})
multisig.braidDetails
multisig.toJSON()
```
or something... again, just spitballing.

Reply:
even this function could be: `multisig.getBip32DerivationByIndex(index)`. But I could also see the benefit of having this be standalone.

Reply:
maps to the JSON.stringify in https://github.com/humanumbrella/unchained-bitcoin/blob/adding-braids/src/braid.js#L86 -- we only sort of define the multisig at this point (agree it should be its own object/struct like extendedpublickey and braid, but it's not right now)